modules/eap-xp-10-env/1.0.0/module.yaml

```diff
@@ -0,0 +1,44 @@
+schema_version: 1
+
+name: "eap-xp-10-env"
+description: "JBoss Enterprise Application Platform XP 1.0 env and labels"
+version: "1.0.0"
+
+labels:
+    - name: "org.jboss.product"
+      value: "eap-xp"
+    - name: "org.jboss.product.version"
```

Comment:
@kabir  out of curiosity, is the XP manager setting these 2 properties (`org.jboss.product` and `org.jboss.product.version`) as System properties or environment variables?